src/sentry/models/integration.py

```diff
@@ -126,9 +127,15 @@ def get_provider(self):
 
         return integrations.get(self.provider)
 
-    def get_installation(self, organization_id, **kwargs):
+    def get_installation(self, organization_id: int, **kwargs: Any) -> Any:
         return self.get_provider().get_installation(self, organization_id, **kwargs)
 
+    def get_installations(self, **kwargs: Any) -> Sequence[Any]:
```

Comment:
Is this used anywhere?

Reply:
`integration.get_installations()` is called in `handle_status_change()`.